Kit/helpers.swift

```diff
@@ -18,7 +18,7 @@ public struct LaunchAtLogin {
     
     public static var isEnabled: Bool {
         get {
-            guard let jobs = (SMCopyAllJobDictionaries(kSMDomainUserLaunchd).takeRetainedValue() as? [[String: AnyObject]]) else {
+                guard let jobs = (LaunchAtLogin.self as DeprecationWarningWorkaround.Type).jobsDict else {
```

Comment:
Thanks, don't know that warning could be hidden. I'm not sure that it's needed, I'm still waiting that Apple will add a new API for this)